js/src/forum/index.js

```diff
@@ -20,31 +20,22 @@ extend(ForumApplication.prototype, 'mount', function() {
   }
 });
 
-m.route.mode = 'pathname';
-
-override(m, 'route', function(original, root, arg1, arg2, vdom) {
-  if (arguments.length === 1) {
-
-  } else if (arguments.length === 4 && typeof arg1 === 'string') {
-
-  } else if (root.addEventListener || root.attachEvent) {
-    root.href = vdom.attrs.href.replace('/embed', '/d');
-    root.target = '_blank';
-
-    // TODO: If href leads to a post within this discussion that we have
-    // already loaded, then scroll to it?
-    return;
-  }
-
-  return original.apply(this, Array.prototype.slice.call(arguments, 1));
+m.route.prefix = '';
+
+override(m.route.Link, 'view', function (original, vnode) {
+  vnode.attrs.href = vnode.attrs.href.replace('/embed', '/d');
+  vnode.attrs.target = '_blank';
+  // TODO: If href leads to a post within this discussion that we have
+  // already loaded, then scroll to it?
+  return original(vnode);
 });
 
 // Trim the /embed prefix off of post permalinks
 override(PostMeta.prototype, 'getPermalink', (original, post) => {
```

Comment:
I believe we can actually remove this. The way the permalink is built uses `/d/` directly.

It's not really specific to this PR and isn't new in beta14 but I figured we might as well do it here.

I've never overriden `getPermalink` in the Wordpress extension and there has never been any issue with permalinks.